cpp/ql/src/semmle/code/cpp/rangeanalysis/RangeAnalysis.qll

```diff
@@ -564,6 +549,7 @@ private predicate boundedInstruction(
     boundedPhiCandValidForEdge(i, b, delta, upper, fromBackEdge, origdelta, reason, op)
   )
   or
+  not i instanceof PhiInstruction and
```

Comment:
Uhm, is this right? We need the base case of a phi variable being bounded by itself.

Reply:
Fixed